src/app/compare/compare-tree/compare-tree.component.ts

```diff
@@ -13,23 +13,28 @@ export class CompareTreeComponent {
   @Output() compareEvent = new EventEmitter<any>();
   @ViewChild('leftTreeReference') leftTreeReference!: jqxTreeComponent;
   @ViewChild('rightTreeReference') rightTreeReference!: jqxTreeComponent;
+  syncTrees: boolean = true;
+  leftReport: any;
+  rightReport: any;
```

Comment:
Ik vind dit niet kunnen, maar goed dan zijn we lang bezig of niet?

Reply:
Dan zouden er Types voor gemaakt moeten worden